ch_tools/monrun_checks/ch_replication_sync.py

```diff
@@ -0,0 +1,42 @@
+import time
+
+from click import command, option, pass_context
+
+from ch_tools.common.result import Result
+from ch_tools.monrun_checks.ch_replication_lag import estimate_replication_lag
+
+
+@command("wait-replication-sync")
+@option(
+    "-s",
+    "--status",
+    type=int,
+    default=0,
+    help="Wait until returned status is no worse than given, 0 = OK (default), 1 = WARN, 2 = CRIT.",
+)
+@option(
+    "-p",
+    "--pause",
+    type=int,
+    default=30,
+    help="Pause between request in seconds, default is 30 seconds.",
+)
+@option(
+    "-t",
+    "--timeout",
+    type=int,
```

Comment:
Suggest to use `TimeSpanParamType` 

Usage example:
https://github.com/yandex/ch-tools/blob/baaf84329cb7f524eef08e8256c6537879934d81/ch_tools/chadmin/cli/object_storage_group.py#L123-L131